services/core-api/app/api/projects/response_models.py

```diff
@@ -214,3 +214,32 @@ def format(self, value):
         'create_user': fields.String,
         'create_timestamp': fields.DateTime
     })
+
+PROJECT_MINE_LIST_MODEL = api.model(
+    'ProjectMineList',{
+        'stage': fields.String,
+        'id': fields.Integer,
+        'guid': fields.String,
+        'project_title': fields.String,
+        'project_id': fields.String,
+        'project_guid': fields.String,
+        'mrc_review_required': fields.Boolean,
+        'status_code': fields.String,
+        'contacts': fields.List(fields.Nested(PROJECT_SUMMARY_CONTACT_MODEL)),
```

Comment:
There is a PROJECT_CONTACT_MODEL I believe that should be used here. I think the PS Contact has been retired, but would need to double check that.

Reply:
Agree, contacts has to be the same as PROJECT_MODEL.